### ISSUE TITLE
Upright partial symbol still looks slightly italic.

### ISSUE THREAD

**Issue by MathStudent999** [alerque/libertinus#193]
While it is less slanted than the italic partial,  partial=upright in unicode-math still gives a slightly slanted partial.
![partial](https://user-images.githubusercontent.com/40512647/41806238-0da8c21c-7688-11e8-9d62-48c0684152f2.png)

**Comment by kauesena** [comment]
I agree the upright partial was indeed only a little less slanted then the italic one and not properly upright. But somehow that was just what a I liked about the old upright version. It would be nice to have the option to use the old glyph.

**Comment by alerque** [comment]
@kauesena I'd be willing to consider adding it back as an alternative, but only if there is some precedent for the style. Your own comment suggests the "proper" thing to do here is have a fully upright Roman glyph design, so where does the other style come into play? Are there other *non italic* glyphs that are commonly drawn partially slanted? Is there precedent for this in other math fonts?

**Comment by kauesena** [comment]
I am afraid I do not know of any font which would constitute such a precedent. It makes sense to condition creating such an alternative to having such a precedent, since it is not a distinctive style alteration such as the ones already implemented. I produced a picture below in order to compare the glyphs.  
![partial symbols comparison](https://user-images.githubusercontent.com/30255100/99720494-9647ad80-2a8c-11eb-9215-646a6030cdd1.png)
In the first line is the italic` \partial`, in the second is the old upright `\partial `and in the third is the current upright `\partial`. There is something in the third character that bugs me. I will keep importing the old character and in the future perhaps try and adjust the current one. If I end up with a character that looks fully upright and feels nicer then the current one, I'll show it here and suggest the replacement.

**Comment by alerque** [comment]
It does look a little off. It appears to be drawn with stroke weight variations suitable to an Italic glyph, but rotated into an upright Roman position. Or maybe it's just the other Italic elements like rounded stroke termination instead of a serif that one expects to define Italic styles that throws your eye off.

I'm not saying the glyph can't be improved and I'd be willing to review a redraw that keeps in style with other similar glyphs, but so far I don't see any reason to suggest it shouldn't be upright.

**Comment by kauesena** [comment]
I agree with you, it should be upright. And that is a good way to put it: "trows your eye off".  
![partial symbols comparison 3](https://user-images.githubusercontent.com/30255100/99739342-504e1200-2aab-11eb-9283-563c9d5148d3.png)
The 3rd one (in black) is the current glyph. The 4th one is the old one. Maybe we could benefit from try to mimic the second last (in orange, from dejavu) by increasing the stroke weight close to the "curled-long-form" meets "o-form". Also, perhaps the upper end of the curled-long-form should tend more horizontally, less downwardly. I do not, however, think this end should be made a proper serif as in the case of the dejavu character, nor be thinned as some of the other ones.

**Comment by Crissov** [comment]
Maybe there is also something to learn from the shape of eth ð here.

**Comment by waldyrious** [comment]
>  It appears to be drawn with stroke weight variations suitable to an Italic glyph, but rotated into an upright Roman position.

This to me nails what makes the glyph seem slightly off when upright.

I'd say that, from the images in @kauesena's latest comment, the fourth-to-last one (in brown) most closely resembles what I'd expect from a non-rotated glyph design in Libertinus — i.e., the heavier strokes are along the vertical sides of the glyph, rather than shifted towards a diagonal axis.

**Comment by kauesena** [comment]
The one @waldyrious mentioned (from Termes) does indeed seem like a good model. I think the one from Spivak's proprietary  (oh, Spivak, why?) font MathTime Pro seems also a good inspiration, 
![partial symbols comparison 5](https://user-images.githubusercontent.com/30255100/99803111-9f319100-2b17-11eb-9cdd-7992d4bf4ea2.png)
